lesson_03/homework_03.py

```diff
@@ -0,0 +1,131 @@
+# task 01 == Split the alice_in_wonderland variable so that it spans multiple physical lines
+
+alice_in_wonderland = '''"Would you tell me, please, which way I ought to go from here?"\n
+"That depends a good deal on where you want to get to," said the Cat.\n
+"I don't much care where ——" said Alice.\n
+"Then it doesn't matter which way you go," said the Cat.\n
+"—— so long as I get somewhere," Alice added as an explanation.\n
+"Oh, you're sure to do that," said the Cat, "if you only walk long enough."\n'''
+
+# task 02 == Find and display all single quote characters ("'") in text
+
+single_quote_char = alice_in_wonderland.count("'")
+print(f"Single quote characters in the text: {single_quote_char}\n")
+
+# task 03 == Print the alice_in_wonderland variable
+
+print(alice_in_wonderland)
+
+"""
+    # Tasks 04-10:
+    # Translate tasks from the book "Mathematics, 5th grade"
+    # to the python language and output the answer, so that it was
+    # understandable to a fifth grade child 
+"""
+# task 04
+"""
+The area of the Black Sea is 436,402 km2, and the area of the Azov Sea 
+is 37,800 km2. What is the area of the Black Sea and the Azov Sea together?
+"""
+black_sea_area = 436402
+azov_sea_area = 37800
+seas_total_area = int(black_sea_area + azov_sea_area)
```

Comment:
there is no need in int() function in this case, because anyway the result is already integer